test/integration/dind_test.js

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
For the record, this is fixed... I just move the blank from top of the file down here... in github diff :)